src/mbgl/text/glyph_pbf.hpp

```diff
@@ -21,7 +21,7 @@ class SDFGlyph {
     AlphaImage bitmap;
 
     // Glyph metrics
-    GlyphMetrics metrics;
+    optional<GlyphMetrics> metrics;
```

Comment:
Once `Glyph` is optional, `GlyphMetrics::operator bool()` is unused, so we don't need to make this optional.